### PR TITLE
feat: add shortcuts for buttons

### PR DESCRIPTION
**Pull request by AnnaRybkina**
Background: https://nmp-jira.atlassian.net/jira/software/projects/WARP/boards/15?selectedIssue=WARP-117

Tested it in my test-warp project 😊 

As a consumer I needed to add the following in the uno config 👇 
image

And it worked 😁 

image

### CODE REVIEW

File: shortcuts.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { button } from './index.js';
+
+export const buttons = {
+    'button': button.buttonSecondary,
```
Comment (magnuh) [inline shortcuts.js:4]: Secondary? 🤔

File: shortcuts.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    'button--secondary': button.buttonSecondary,
+    'button--default': button.buttonSecondary,
```
Comment (magnuh) [inline shortcuts.js:6]: Should they all be secondary? 😅

File: shortcuts.js (hunk 1 above)
Comment (AnnaRybkina) [reply shortcuts.js:4]: It is apparently a default, aka secondary button. https://github.com/fabric-ds/css/blob/next/src/components/button.css#L3

File: shortcuts.js (hunk 2 above)
Comment (AnnaRybkina) [reply shortcuts.js:6]: https://github.com/fabric-ds/css/blob/next/src/components/button.css#L3 yes

**Review by siljec — APPROVED** [review]
Cool! LGTM 🎉  
Did we rename button quiet to button flat? I didn't recognise the flat property but I see that it's used when we have quiet props in react at least 👀

**Comment by AnnaRybkina** [comment]
> Cool! LGTM 🎉 Did we rename button quiet to button flat? I didn't recognise the flat property but I see that it's used when we have quiet props in react at least 👀

We didn't rename anything, as far as I remember. It was like that in fabric. `flat` in class name but the prop is `quiet`. https://github.com/fabric-ds/css/blob/next/src/components/button.css#L28

**Pull request merged**